src/factories.py

```diff
@@ -162,6 +162,7 @@ class Meta:
         created_by=factory.SelfAttribute('..owner'),
         created_when=factory.SelfAttribute('..created_when'),
     )
+    task = factory.LazyAttribute(lambda submission: random.choice(submission.phase.tasks.all()))
```

Comment:
Was having problems because my submissions didn't have tasks assigned after a `./reset_db.sh`. This fixes it!